Ginger/GingerCoreNET/Reports/GingerExecutionReport/EmailExecutionReport.html

```diff
@@ -16,7 +16,7 @@
             padding: 0 !important;
             height: 100% !important;
             width: 100% !important;
-            font-family: CenturyGothic;
+            font-family: Arial, Helvetica, sans-serif;
```

Comment:
you need to do same changes also on "EmailExecutionReportOnLinux.HTML" which been used when running from Linux box